recipes/ff/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "ff" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name }}-static
+  version: {{ version }}
+
+source:
+    url: https://github.com/hep-packaging-coordination/ff/archive/v{{ version }}.tar.gz
+    sha256: 2329b3ca147825b660e07dac45ed85d397a1bea2f6b886518eb4d09900da5503
+    patches:
+      # Can be removed after the patched QCDLoop version is merged in v2.0.1.
+      - update-makefile.patch
+
+build:
+  skip: true  # [win]
+  number: 0
+  script:
+    # FIXME: Patch out custom paths
+    - sed -i "s|/user/gj/lib/|$PREFIX/share/ff/|g" src/ff/ffinit.f
+    - sed -i "s|/usr/local/ff/|$PREFIX/share/ff/|g" src/ff/ffinit.f
+    # FIXME: Patch a path character length long enough to handle $PREFIX
+    - sed -i 's/128 path,fullname/4096 path,fullname/g' src/ff/ffinit.f
```

Comment:
Not sure if there is a smarter / more effective way to try to deal with needing to get `$PREFIX`'s very long length during `conda-build` into a variable in Fortran or not.